src/sentry/static/sentry/app/views/settings/organization/members/organizationMembersView.jsx

```diff
@@ -1,6 +1,7 @@
-import {Box} from 'grid-emotion';
 import PropTypes from 'prop-types';
 import React from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
I'm unsure if our lodash plugin will tree-shake this (@MaxBittker do you know?). But I know these do:

```
import {debounce} from 'lodash';
import debounce from 'lodash/debounce';
```